FluentUI.Demo/src/main/res/layout/activity_tooltip.xml

```diff
@@ -58,4 +58,26 @@
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintLeft_toLeftOf="parent" />
 
+    <com.microsoft.fluentui.widget.Button
```

Comment:
Can we check the UI for duo devices? Please test the app on Duo too

Reply:
Removed the AvatarGroup Demo
Here are screenshots on Duo for the latest commit: 

![Screenshot_1632890540](https://user-images.githubusercontent.com/88475350/135204886-d9833f2b-1250-42ac-9e07-d199ceeb4790.png)
![Screenshot_1632890546](https://user-images.githubusercontent.com/88475350/135204891-9f2ccbae-aea4-4f42-8be8-36373089c9dc.png)
![Screenshot_1632890567](https://user-images.githubusercontent.com/88475350/135204895-ebd1b9b3-8988-46b9-9ba7-fd9e69baa6db.png)
![Screenshot_1632890572](https://user-images.githubusercontent.com/88475350/135204898-28016eb5-be16-4431-a13e-2ec5252afaec.png)
![Screenshot_1632890582](https://user-images.githubusercontent.com/88475350/135204899-258f14c9-d7af-4013-b33c-8863ed080db4.png)
![Screenshot_1632890585](https://user-images.githubusercontent.com/88475350/135204901-0d98d99b-4d51-4e29-9829-c97a1b748102.png)